piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -1355,13 +1411,16 @@ sentence of premunire was given against him by the judge, in these words:
 your lands are confiscated to the king during your life,
 and your goods and chattles forever; and you to be a prisoner during your life.`'
 
+[.discourse-part]
 F+++.+++ Howgill replied, '`A hard sentence for my obedience to the commands of Christ;
 the Lord forgive you all.`'
 
-Judge: '`Well, if you will yet be subject to the laws of the king,
+[.discourse-part]
+_Judge:_ '`Well, if you will yet be subject to the laws of the king,
 the king will show you mercy.`'
 
-Francis Howgill: '`The Lord has showed mercy unto me,
+[.discourse-part]
+_Francis Howgill:_ '`The Lord has showed mercy unto me,
 and I have done nothing against the king, or government, or any man, blessed be the Lord,
 and herein stands my peace; for it is for Christ`'s sake I suffer,
 and not for evil doing.`'
```

Comment:
I would put a small-break after this line too, so that there is a space between Howgill's comment and where the narrator picks back up.